i18n/zh-CN/docusaurus-plugin-content-blog/2023-11-30-kcl-0.7.0-release/index.md

```diff
@@ -0,0 +1,206 @@
+
+#### lambda 表达式类型注解
+
+新增了 lambda 表达式的类型注解，您可以在新版本的 KCL 中编写如下带有类型注解的 lambda。
+
+```
+import sub as s
+
+identity: (s.Test) -> s.Test = lambda res: s.Test {
+  res {
+    name = "world"
+  }
+}
+
+c = identity(s.Test{name="hello"})
+```
+
+### 🏄 API 更新
+
+- 新增 KCL 单元测试 API: https://github.com/kcl-lang/kcl/pull/904
+- KCL Schema 模型解析API增强版 GetFullSchemaType支持获取带有三方库的 KCL 包相关信息和 Schema 属性默认值 https://github.com/kcl-lang/kcl/pull/914
+
+### 🐞 错误修复
+
+- 修复部分场景字典类型的类型推导错误 https://github.com/kcl-lang/kcl/pull/900
+- 修复 Schema 参数的检查 https://github.com/kcl-lang/kcl/pull/877/files
+- 修复了带有三方库的KCL程序编译缓存失效的问题 https://github.com/kcl-lang/kcl/pull/841
+- 错误信息中补全缺失的 lambda 的类型信息 https://github.com/kcl-lang/kcl/pull/771
+- 修复了诊断信息中单数和复数的问题 https://github.com/kcl-lang/kcl/pull/769
+- 修复了带有类型注解赋值语句中类型检查失效的问题 https://github.com/kcl-lang/kcl/pull/757
+
+- 增加了检查，禁止同名的 import 语句 https://github.com/kcl-lang/kcl/pull/727
+
+## 工具链更新
+
+### 测试工具更新
+
+担心您的 KCL 程序写错了，要不来测测 ？本次更新提供了全新的 KCL 测试工具，代码好坏，一测便知 ！新的 KCL  测试工具支持使用 KCL 函数编写单元测试并使用工具执行测试。
+
+您可以在后缀名为 “_test.k” 文件中通过 lambda 表达式来编写您的测试用例。
+
+```
+import manifests
+
+import .app
+
+# Convert the `App` model into Kubernetes Deployment and Service Manifests
+test_kubernetesRender = lambda {
+    a = app.App {
+        name = "app"
+        containers.ngnix = {
+            image = "ngnix"
+            ports = [{containerPort = 80}]
+        }
+        service.ports = [{ port = 80 }]
+    }
+    deployment_got = kubernetesRender(a)
+    assert deployment_got[0].kind == "Deployment"
+    assert deployment_got[1].kind == "Service"
+}
+```
+
+通过 kcl test 您可以运行这个测试用例并查看测试结果。
+
+测试通过您将得到如下结果：
+
+![test-pass](/img/blog/2023-11-30-kcl-0.7.0-release/test-pass.png)
+
+如果您的测试失败了，kcl test 将输出错误信息帮助您排查问题所在。
+
+![test-failed](/img/blog/2023-11-30-kcl-0.7.0-release/test-failed.png)
+
+### KCL 包管理
+
+新增 update 命令用于自动更新本地依赖，kcl mod update 将会为您自动下载本地缺失的三方库。具体参考: https://github.com/kcl-lang/kpm/pull/212
+
+## 社区集成 & 扩展更新
+
+### KCL 三方库市场 artifacthub.io
+
+通过集成 artifacthub.io 我们构建了一个 KCL 三方库市场，在这里您尽可以大展您的才华，与我们一同分享您对 KCL 程序的独特理解，您也可以尽情挑选，找到适合您自己的 KCL 三方库 ！
+
+打开首页，直接搜索您需要的关键字，就能看到关于 KCL 三方库的相关内容 ！
+
+![artifachub-index](/img/blog/2023-11-30-kcl-0.7.0-release/artifachub-index.png)
+
+在三方库首页，可以查看关于三方库的详细内容和相关文档。
+
+![pkg-page](/img/blog/2023-11-30-kcl-0.7.0-release/pkg-page.png)
+
+如果您不知道该如何使用这些三方库，右侧的按钮可以为您唤起安装页面。
+
+![install-pkg](/img/blog/2023-11-30-kcl-0.7.0-release/install-pkg.png)
+
+## 其他更新
+
+完整更新和错误修复列表详见: https://github.com/kcl-lang/kcl/compare/v0.6.0...v0.7.0
+
+## 文档更新
+
+KCL 网站 新增 KCL v0.7.0 文档内容并支持版本化语义选项，目前支持 v0.4.x, v0.5.x 和 v0.7.0 版本选择，同时欢迎社区同学进行文档共建。
+
+## 社区动态
+
+- 感谢 @XiaoK29 对 IDE 悬停功能体验提升的贡献 🙌
+- 感谢 @jakezhu9 对编译器单元测试，benchmark 的贡献 🙌
+
+## 下一步计划
+
+预计 2023 年 11 月，我们将发布 KCL v0.8.0 版本，更多详情请参考 KCL 2024 路线规划 和 KCL v0.8.0 Milestone，如果您有更多的想法和需求，欢迎在 KCL Github 仓库发起 Issues 或讨论，也欢迎加入我们的社区进行交流 🙌 🙌 🙌
+- KCL 2024 路线规划: https://kcl-lang.io/docs/community/release-policy/roadmap
```

Comment:
Roadmap is here: https://github.com/kcl-lang/kcl/issues/882

Reply:
fixed.